docs/tutorial/7-add-interactivity.md

```diff
@@ -104,6 +103,8 @@ window.addEventListener('load', async () => {
 });
 ```
 
+![The terminal window now shows the typical terminal prompt](./images/19-terminal-prompt.png)
```

Comment:
I moved the image because you only see the output as shown in the image after the previous code block. Unless there was a reason to move it before that codeblock?

![image](https://user-images.githubusercontent.com/1913805/221125072-f92cb4ee-0707-438d-acf6-ef3e192f4d1f.png)

Reply:
Thank you! This is the correct placement!